nservicebus/security/encryption.md

```diff
@@ -96,12 +108,19 @@ This allows you to explicitly handled the encryption and decryption of each valu
 
 ## Message Encryption
 
-Message encryption leverages the pipeline to apply encryption to the whole message body.
+Message encryption encrypts the complete message body. It can even include the message headers if these would contain sensitive data.
```

Comment:
Perhaps better in a SC caveats page and link to from here. Also SC doesnt support custom serializers either.